Polyhedron/demo/Polyhedron/MainWindow.cpp

```diff
@@ -1015,7 +1015,7 @@ void MainWindow::open(QString filename)
     Q_FOREACH(CGAL::Three::Polyhedron_demo_io_plugin_interface* io_plugin, io_plugins) {
       if ( !io_plugin->canLoad() ) continue;
       all_items << io_plugin->name();
-      if ( file_matches_filter(io_plugin->loadNameFilters(), filename) )
+      if ( file_matches_filter(io_plugin->loadNameFilters().toLower(), filename) )
```

Comment:
Wrong patch!

That is not the filters you want to `toLower()`, but the file names! Here:

https://github.com/afabri/cgal/blob/0caaf4136e5f8673bad64027a38410d6c25c3f6c/Polyhedron/demo/Polyhedron/MainWindow.cpp#L937-L960

L953 the string `filename_striped` should be `toLower()` before being matched.